packages/kbn-securitysolution-list-utils/src/helpers/index.ts

```diff
@@ -696,8 +698,12 @@ export const getOperatorOptions = (
 ): OperatorOption[] => {
   if (item.nested === 'parent' || item.field == null) {
     return [isOperator];
-  } else if ((item.nested != null && listType === 'endpoint') || listType === 'endpoint') {
-    return isBoolean ? [isOperator] : [isOperator, isOneOfOperator];
+  } else if (listType === 'endpoint') {
+    return isBoolean
+      ? [isOperator]
+      : fieldSupportsMatches(item.field)
```

Comment:
The [kibana style guide](https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx) recommends against nested ternaries like this; it does make things a bit hard to parse, here.